medical/tests/test_medical_patient.py

```diff
@@ -139,3 +139,20 @@ def test_check_birthdate_date(self):
                     now + timedelta(days=20),
                 )
             })
+
+    def test_search_age(self):
```

Comment:
This is a false positive. You should instead perform a search on age and make sure the right patients are returned